src/components/cardList/CardList.jsx

```diff
@@ -119,12 +119,14 @@ class CardList extends React.Component {
       <div>
         <h3><Label>{this.props.title}</Label></h3>
         <h3><Label>{''}</Label></h3>
-        {this.props.filters &&
+        {/* Temporarily commenting this out till I understand what should happen to this filter */}
+        {/* {this.props.filters &&
           <div className="filters">
             <span>Filter:</span>
             <span>{ this.props.optionalFilters ? filterButtonToolbar : undefined }</span>
           </div>
-        }
+        } */}
+        {this.props.cardListFilters && this.props.cardListFilters()}
```

Comment:
What is the purpose of this new cardListFilters prop?  Is this a work-in-progress?